go/cache/lru_cache_test.go

```diff
@@ -26,24 +26,24 @@ type CacheValue struct {
 	size int
 }
 
-func (cv *CacheValue) Size() int {
+func (cv *CacheValue) CachedSize() int {
 	return cv.size
 }
 
 func TestInitialState(t *testing.T) {
 	cache := NewLRUCache(5)
-	l, sz, c, e, _ := cache.Stats()
-	if l != 0 {
-		t.Errorf("length = %v, want 0", l)
+	stats := cache.Stats()
+	if stats.Length != 0 {
```

Comment:
nit: we try to move to testify whenever we have to change these types of assertions in the code

Reply:
SGTM. I've noticed that we're mixing testify and classic assertions. I'll update the tests. :+1: